src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.CSharpCodeGenerator.CallSiteContainerRewriter.cs

```diff
@@ -61,9 +61,13 @@ public override SyntaxNode VisitLocalDeclarationStatement(LocalDeclarationStatem
                 {
                     node = (LocalDeclarationStatementSyntax)base.VisitLocalDeclarationStatement(node);
 
+                    if (node.Declaration.IsDeconstructionDeclaration)
+                    {// PROTOTYPE(tuples)
```

Comment:
@CyrusNajmabadi I intend to remove this PROTOTYPE marker before checking in. The question here is whether this behavior is sufficient, or if I should add logic for removing variables.
I'm not familiar enough with scenarios involving removing variables, so I'm not sure if it could be applicable to deconstructions. I'm guessing it's not applicable.